modules/machine-pools-hcp.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * rosa_cluster_admin/rosa_nodes/rosa-nodes-machinepools-about.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="machine-pools-hcp_{context}"]
+= Machine pools in {hcp-title} clusters
+
+In {hcp-title} clusters, the {hcp} spans three availability zones (AZ) in the installed cloud region. Each machine pool in a {hcp-title} cluster deploys in a single subnet within a single AZ. Each of these AZs can only have one machine pool. 
+
+Each machine pool in an {hcp-title} cluster upgrades independently. Because the machine pools upgrade independently, they must remain within 2 minor (Y-stream) versions of the hosted control plane. For example, if your hosted control plane is 4.16.z, your machine pools must be at least 4.14.z.
+
+The following images depict how machine pools work within ROSA and {hcp-title} clusters:
```

Comment:
Nit - is this technically just one image? Or are you referring to the fact that this image file has 3 separate sections of diagrams?
```suggestion
The following image depicts how machine pools work within ROSA and {hcp-title} clusters:
```

Ignore this if your wording was intentional